api/v1alpha1/externaldns_webhook.go

```diff
@@ -134,6 +134,9 @@ func (r *ExternalDNS) validateFilters() error {
 }
 
 func (r *ExternalDNS) validateHostnameAnnotationPolicy() error {
+	if r.Spec.Source.Type == SourceTypeRoute {
+		return nil
+	}
```

Comment:
This fixes the e2e-gcp tests ?

Reply:
yes, but actually I should revert this and all the tests .. There is a validationWebhook tests for this, and I should have kept the FQDN template 

fix on the way